src/calls/components/CallsView/CallsView.jsx

```diff
@@ -0,0 +1,149 @@
+                volonter: 'Marko'
+            },
+            {
+                id: 3,
+                time: '12/11/2011',
+                duration: 35,
+                person: 'Naca',
+                type: 'Hitan slucaj',
+                risk: 'srednji',
+                volonter: 'Ljilja'
+            },
+            {
+                id: 4,
+                time: '12/11/2031',
+                duration: 35,
+                person: 'Zaca',
+                type: 'Hitan slucaj',
+                risk: 'srednji',
+                volonter: 'Ljilja'
+            },
+            {
+                id: 5,
+                time: '24/11/2019',
+                duration: 35,
+                person: 'Kaca',
+                type: 'Hitan slucaj',
+                risk: 'srednji',
+                volonter: 'Ljilja'
+            }
+        ];
+        this.setState({ initialItems: items });
+    };
+
+    handleChangeTableData = date => {
+        let items = this.state.initialItems;
+        let itemsByDate = items.filter(p => {
+            return p.time === format(date, 'dd/MM/yyyy').toString();
+        });
+
+        this.setState({
+            items: [...itemsByDate]
+        });
+    };
+    renderTableData() {
+        return this.state.items.map((item, index) => {
+            const {
+                id,
+                time,
+                duration,
+                person,
+                type,
+                risk,
+                volonter
+            } = item; //destructuring
+            return (
+                <tr key={id}>
+                    <td>{id}</td>
+                    <td>{time}</td>
+                    <td>{duration}</td>
+                    <td>{person}</td>
+                    <td>{type}</td>
+                    <td>{risk}</td>
+                    <td>{volonter} </td>
+                </tr>
+            );
+        });
+    }
+
+    render() {
+        return (
+            <div className="container-fluid col-lg-12">
+                <div className="col-lg-2 d-inline-block button-group">
+                    <button
+                        type="button"
+                        className="btn btn-primary d-inline-block btn-sm"
+                    >
+                        Add
+                    </button>
+                    <button
+                        type="button"
+                        className="btn btn-danger d-inline-block btn-sm"
+                    >
+                        Delete
+                    </button>
+                    <button
+                        type="button"
+                        className="btn btn-success d-inline-block btn-sm"
+                    >
+                        Details
+                    </button>
+                </div>
+
+                <div className="col-lg-6 d-inline-block callendarSpace">
+                    <span>Kalendar</span>
+                    <Calendar
+                        handleChangeTableData={date =>
```

Comment:
It's not clear from this prop name when it is triggered. Like `onClick`, `onMouseOver`, and so on, consider calling this prop something like `onDateChange`, or `onSelect`, or `onDateSelect`, or something...